src/pages/guides/using/asset-events/asset-events-configuration.md

```diff
@@ -0,0 +1,21 @@
+# Asset Events Configuration
+
+To configure Asset Events, refer to [Add Events]((/apis/experienceplatform/console/docs.html#!AdobeDocs/adobeio-console/master/services-add-event.md)), keeping in mind the following amendments to the **Add Events** and **Credentials** sections:
```

Comment:
The link for `Add Events` is not correctly configured here. This needs to be absolute link to https://www.adobe.io/apis/experienceplatform/console/docs.html#!AdobeDocs/adobeio-console/master/services-add-event.md.

Reply:
done